docs/src/assets/readme.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+In this folder we store the following data sets:
+
+- german_credit.csv
+- iris.csv
+
+# German Credit data set
+
+## License:
+https://opendatacommons.org/licenses/dbcl/1-0/
+
+## Source:
+https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)
+Professor Dr. Hans Hofmann
+Institut f"ur Statistik und "Okonometrie
+Universit"at Hamburg
+FB Wirtschaftswissenschaften
+Von-Melle-Park 5
+2000 Hamburg 13
+
+The original data is from [UCI](https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)), and that this data is from [Kaggle](https://www.kaggle.com/uciml/german-credit)
+
+
```

Comment:
```suggestion
# Iris data set

## License
https://creativecommons.org/publicdomain/zero/1.0/

## Source:
https://archive.ics.uci.edu/ml/datasets/Iris
Creator:
R.A. Fisher
```

Reply:
in the future you can just clik "Commit suggestion" (or edit it before accepting and then commit)